server/src/test/scala/services/WorkSpec.scala

```diff
@@ -0,0 +1,7 @@
+package services
+
+import org.specs2.mutable.Specification
+
+class WorkSpec extends Specification {
```

Comment:
I guess you don't need this one or need to add test

Reply:
Good catch. I had missed a bunch of tests. I've added them now